adagucserverEC/CDBAdapterPostgreSQL.h

```diff
@@ -43,6 +43,8 @@ class CDBAdapterPostgreSQL : public CDBAdapter {
   std::map<std::string, std::vector<std::string>> fileListPerTable;
   int createDimTableOfType(const char *dimname, const char *tablename, int type);
 
+  CDBStore::Store *layerMetaDataStore = nullptr;
```

Comment:
This file has no `#ifndef` guards. Same for `CDBAdapterSQLLite.h`. They feel left out.

Reply:
I hope that they are happy with their guards now ;)